UPGRADE-5.0.md

```diff
@@ -14,6 +14,19 @@ UPGRADE FROM 4.x to 5.0
   used directly or extended.
 * Remove support of Annotation mapping, you should use Attributes or XML instead.
 
+## Configuration
+
+Deprecated options have been removed:
+
+| namespace                | removed     | replaced by        |
+|--------------------------|-------------|--------------------|
+| `default_commit_options` | `fsync`     | `j`                |
+| `default_commit_options` | `safe`      | `w`                |
+| `connections.*.options`  | `fsync`     | `journal`          |
```

Comment:
I'm not sure if the `.*.` is the correct notation we use to reference paths with variable parts.